sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/aio/_operations/_patch.py

```diff
@@ -16,14 +16,13 @@
 from azure.core.polling import AsyncLROPoller, AsyncNoPolling, AsyncPollingMethod
 
 from azure.confidentialledger.aio._operations._operations import (
-    ConfidentialLedgerClientOperationsMixin as GeneratedOperationsMixin,
+    _ConfidentialLedgerClientOperationsMixin as GeneratedOperationsMixin,
 )
 from azure.confidentialledger.aio._operations._operations import ClsType, JSON
 from azure.confidentialledger._operations._patch import BaseStatePollingMethod
+import azure.confidentialledger.models as _models
 
-__all__: List[str] = [
-    "ConfidentialLedgerClientOperationsMixin"
-]  # Add all objects you want publicly available to users at this package level
```

Comment:
same as https://github.com/Azure/azure-sdk-for-python/pull/41924/files#r2274917088